app/models/collection.rb

```diff
@@ -74,7 +81,6 @@ def delete_permission(current_user, removed_user)
         errors.add(:delete_permission, "Cannot remove yourself from a collection. Contact a super-admin for help.")
       else
         errors.delete(:delete_permission)
-        removed_user.remove_role :collection_admin, self
```

Comment:
Why did this get changed in this PR?

Reply:
I am sorry, this must have been a rebase error. This has been fixed.